### ISSUE TITLE
swashbuckle quickcast ability changed

### ISSUE THREAD

**Issue by ShayyahS** [ValveSoftware/Dota2-Gameplay#4632]
### Description

As I recall, swashbuckler when casted using quickcast with 'quickcast on key down' setting enabled would function like so:

- Press Q (press key down and let go) to choose the point to jump to & pick the direction of slashing by moving the mouse and either left clicking or pressing Q again

Now it works like so:

- Press Q (press down and let go) and the ability instantly casts giving no time to aim the slashes.

Please could the old functionality of the ability be added back as an option? I struggle to play pango now because of this huge change to his mechanics and he is my most played hero that I would go to as a way to ease myself back into the game but the core ability is completely different.

**Comment by Onionscale** [comment]
Yes, now quickcast on key down acts like quickcast on key up, making it useless for vector target spells. Please fix!

**Comment by Onionscale** [comment]
Issue still persists - quickcast on key down and key up act the same way for vector targeted spells, which is not good. Please revert quickcast on key down to necessitate a left click for vector targetted spells.
